src/utils/index.ts

```diff
@@ -147,3 +145,57 @@ export const getRouletteTableInfo = ({
 
   return gameInfos;
 };
+
+function between(x: number, min: number, max: number) {
+  return x >= min && x <= max;
+}
+
+export const checkComputerBet = (outcome: number) => {
```

Comment:
This is the correct math, but how about for the game partners? We have a different edge on the partner nfts